tasks/main.yml

```diff
@@ -8,7 +8,7 @@
 
 - name: Set Certbot package name and versions (Ubuntu < 20.04)
   set_fact:
-    certbot_version: 0.31.0-1+ubuntu{{ ansible_distribution_version }}.1+certbot+1
+    certbot_version: 0.31.0-2~deb10u1+ubuntu{{ ansible_distribution_version }}.1+certbot+3
```

Comment:
I just confirmed this change is needed on Ubuntu 18, but the old version is need for Ubuntu 16...

Reply:
Sorry, my mistake. The new version works on Ubuntu 16 as well :+1:

Reply:
Whew 😅